boards/arm/nrf53/nrf5340-dk/src/nrf53_bringup.c

```diff
@@ -32,21 +33,122 @@
 #ifdef CONFIG_USERLED
 #  include <nuttx/leds/userled.h>
 #endif
+#ifdef CONFIG_RPMSG_UART
+#  include <nuttx/serial/uart_rpmsg.h>
+#endif
 
 #ifdef CONFIG_NRF53_SOFTDEVICE_CONTROLLER
 #  include "nrf53_sdc.h"
 #endif
 
 #ifdef CONFIG_RPTUN
+#  include <nuttx/wireless/bluetooth/bt_rpmsghci.h>
+#  ifdef CONFIG_UART_BTH4
+#    include <nuttx/serial/uart_bth4.h>
+#  endif
+#  ifdef CONFIG_NET_BLUETOOTH
+#    include <nuttx/wireless/bluetooth/bt_driver.h>
+#  endif
 #  include "nrf53_rptun.h"
 #endif
 
 #include "nrf5340-dk.h"
 
+/****************************************************************************
+ * Private Functions
+ ****************************************************************************/
+
+#ifdef CONFIG_NRF53_APPCORE
+
+/****************************************************************************
+ * Name: nrf53_appcore_bleinit
+ ****************************************************************************/
+
+static int nrf53_appcore_bleinit(void)
+{
+  struct bt_driver_s *bt_dev = NULL;
+  int                 ret    = OK;
+
+#ifdef CONFIG_BLUETOOTH_RPMSG
+  bt_dev = rpmsghci_register("appcore", "bthci");
+  if (bt_dev == NULL)
+    {
+      syslog(LOG_ERR, "ERROR: rpmsghci_register() failed: %d\n", -errno);
+      goto errout;
```

Comment:
```suggestion
  return -ENOMEM;
```
and remove errout label

Reply:
done